src/js/components/CombinationMenuItem.vue

```diff
@@ -4,27 +4,18 @@
     <i class="fa fa-clone" @click="cloneScroll"></i>
     <i v-show="locked" class="fa fa-lock" style="color: red"></i>
     <i v-show="!locked" class="fa fa-unlock" style="color: green"></i>
-    <!-- <div v-if="open">
-      <el-radio-group v-model="menuType" size="mini">
-        <el-radio-button label="art"></el-radio-button>
-        <el-radio-button label="col"></el-radio-button>
-        <el-radio-button label="img"></el-radio-button>
-      </el-radio-group>
-    </div> -->
     <div class="children" v-show="open">
         <ul>
-          <!-- <li v-show="menuType === 'art'" v-for="child in children">
-            <art-menu-item :data-id="child.id"></art-menu-item>
-          </li> -->
-          <li v-show="menuType === 'col'" v-for="child in children">
-            <col-menu-item :data-id="child.id"
-                            :name="child.name"></col-menu-item>
+          <li v-if="menuType === 'text'" v-for="child in children">
+            <column-menu-item :data-id="child.id"
+                            :name="child.name"></column-menu-item>
           </li>
-          <li v-show="menuType === 'img'" v-for="child in children">
-            <img-menu-item :data-id="child.id"
+          <li v-if="menuType === 'image'" v-for="child in children">
+            <image-menu-item :data-id="child.id"
```

Comment:
Not important now, but my preference for formatting custom components where there's lots of custom vue attributes is like this:

```html
<image-menu-item
  :data-id="child.id"
  :institution="child.institution"
  ... etc
>
  <stuff in slots/>
</image-menu-item>
```
For standard html where we just have class, id, and whatnot, I find a single line easier to read:
```html
<div id="some-id" class="some-class">
 <!-- etc -->
</div>
```

Reply:
I like the suggestion, it looks more readable that way.